app/src/test/java/DataBinderMapperImpl.java

```diff
@@ -0,0 +1,7 @@
+package androidx.databinding;
+
+public class DataBinderMapperImpl extends MergedDataBinderMapper {
```

Comment:
public final class

Also, definitely add documentation explaining why this is needed and a TODO to remove it once android_local_test properly supports data binding.

Reply:
Done